### PR TITLE
Inquiry list view added

### PR DESCRIPTION
**Pull request by keshavgr8**
Finishes story [#154697533](https://www.pivotaltracker.com/story/show/154697533)
inq-list-viewpr

### CODE REVIEW

File: client-src/src/pages/inq-list/inq-list.html [hunk 1]
```diff
@@ -11,37 +11,91 @@
...
+  <ion-card *ngFor = "let inq of inquiries; let i = index;">
+    <ion-grid>
+      <ion-row>
+        <ion-col col-1 class="vertical-center-contents">
+          <p>{{i+1}}.</p>
+        </ion-col>
+        <ion-col col-1 class="vertical-center-contents">
+          <div *ngIf = "inq.caseIndex">
+            <div *ngIf = "inq.caseIndex.index == 3">
+              <ion-icon name="flash" style="color:#ffd633"></ion-icon>
```
Comment (aayushKhandpur) [inline client-src/src/pages/inq-list/inq-list.html]: Can we avoid inline css?

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review

File: client-src/src/pages/inq-list/inq-list.html (hunk 1 above)
Comment (keshavgr8) [reply client-src/src/pages/inq-list/inq-list.html]: fixed in commit e00635f

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Resolve conflicts.

**Comment by keshavgr8** [comment]
Conflicts resolved

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**